src/H5TSprivate.h

```diff
@@ -332,24 +380,24 @@ typedef struct H5TS_pt_rec_rw_lock_stats_t {
     int64_t write_locks_delayed;
     int64_t max_write_locks_pending;
 
-} H5TS_pt_rec_rw_lock_stats_t;
+} H5TS_rw_lock_stats_t;
 
 /******************************************************************************
  *
- * Structure H5TS_pt_rec_rw_lock_t
+ * Structure H5TS_rw_lock_t
  *
  * A read / write lock, is a lock that allows either an arbitrary number
  * of readers, or a single writer into a critical region.  A recurssive
  * lock is one that allows a thread that already has a lock (be it read or
  * write) to successfully request the lock again, only droping the lock
```

Comment:
Typo here - missing second p in droping  - could be fixed while changes are being made to the file.

Reply:
Fixed